{"name": "test_login_propio", "status": "failed", "statusDetails": {"message": "AssertionError: El rol no corresponden al usuario\nassert 'Administrador' in 'ADMINISTRADOR'", "trace": "setup_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"4f6919a0b854846b2db913a571037508\")>\n\n    @allure.feature('Login Tests')\n    def test_login_propio(setup_browser):\n        driver = setup_browser\n    \n        try:\n    \n            #Confirnmación de pagina colegium\n            assert \"Colegium\" in driver.title, \"Título incorrecto de la página\"\n            time.sleep(3)\n    \n            # Ingresar usuario y contraseña\n            Email = driver.find_element(By.XPATH, \"/html[1]/body[1]/section[1]/div[2]/div[1]/div[1]/form[1]/div[1]/input[1]\")\n            Email.send_keys(\"pormero@colegium.com\")\n            time.sleep(1)\n            Password = driver.find_element(By.XPATH, \"/html[1]/body[1]/section[1]/div[2]/div[1]/div[1]/form[1]/div[2]/input[1]\")\n            Password.send_keys(\"Pop23725\")\n            time.sleep(1)\n            Password.send_keys(Keys.RETURN)\n            time.sleep(1)\n            driver.find_element(By.XPATH, \"//button[@type='submit']\")\n            time.sleep(1)\n    \n            # Verificar que se haya iniciado sesión correctamente\n            Colegio = driver.find_element(By.XPATH, \"/html/body/nav/div/div[1]/div/p\")\n            Colegio_text = Colegio.text\n            assert \"Colegio Aleman De La UniÓn\" in Colegio_text, \"Colegio Erroneo\"\n            Usuario = driver.find_element(By.CSS_SELECTOR, \"p[class='mb-0 title']\")\n            Usuario_text = Usuario.text\n            assert \"Pablo Daniel Ormero\" in Usuario_text, \"Nombre de usuario incorrecto\"\n            Rol  = driver.find_element(By.CSS_SELECTOR, \"span[class='mb-0 sub-title']\")\n            Rol_text = Rol.text\n>           assert \"Administrador\" in Rol_text, \"El rol no corresponden al usuario\"\nE           AssertionError: El rol no corresponden al usuario\nE           assert 'Administrador' in 'ADMINISTRADOR'\n\ntest_2_LoginPropio.py:80: AssertionError"}, "attachments": [{"name": "screenshot_error", "source": "5810ade9-5b71-4a2b-9aaa-34d85cb69e08-attachment.png", "type": "image/png"}, {"name": "stdout", "source": "8176ce10-4347-4d76-963e-d65f6766dda9-attachment.txt", "type": "text/plain"}], "start": 1723249732091, "stop": 1723249740643, "uuid": "687ad03c-e101-462e-b8fc-d252af5bbdca", "historyId": "65d066257009b9bf5df27da262f4612c", "testCaseId": "65d066257009b9bf5df27da262f4612c", "fullName": "test_2_LoginPropio#test_login_propio", "labels": [{"name": "feature", "value": "Login Tests"}, {"name": "suite", "value": "test_2_LoginPropio"}, {"name": "host", "value": "DESKTOP-6A7N4QP"}, {"name": "thread", "value": "27400-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_2_LoginPropio"}]}